.circleci/config.yml

```diff
@@ -186,21 +178,12 @@ commands:
       architecture:
         type: string
         default: "amd64"
-      cuda-pkg-name:
-        type: string
-        default: "cuda-toolkit-11-8"
-      cuda-pkg-version:
-        type: string
-        default: "11-8"
-      cuda-version:
-        type: string
-        default: "11.8.0"
       cuda-string-version:
         type: string
-        default: "cuda11.8"
+        default: "cuda12.0"
```

Comment:
This version is `cuda12.0` and not `cuda12.1` like the others because the `apt` Ubuntu package directory does not have `nvinfer` libraries specifically for `cuda12.1`.